backend/grpc-server/src/http/router.rs

```diff
@@ -0,0 +1,68 @@
+use axum::{
+    routing::{get, post},
+    Router,
+};
+
+use super::{handlers, state::AppState};
+
+pub fn create_router(state: AppState) -> Router {
+    Router::new()
+        .route("/health", get(handlers::health))
+        .route("/payments/authorize", post(handlers::payments::authorize))
+        .route(
+            "/payments/authorize_only",
+            post(handlers::payments::authorize_only),
+        )
+        .route("/payments/capture", post(handlers::payments::capture))
+        .route("/payments/void", post(handlers::payments::void))
+        .route(
+            "/payments/void_post_capture",
+            post(handlers::payments::void_post_capture),
+        )
```

Comment:
The `/payments/get` endpoint uses POST method but the operation is a read/query. Consider using GET method instead for semantic correctness:

```suggestion
        .route("/payments/get", get(handlers::payments::get_payment))
```